scripts/irods/test/test_imiscsvrinfo.py

```diff
@@ -0,0 +1,32 @@
+import sys
+
+if sys.version_info < (2, 7):
+    import unittest2 as unittest
+else:
+    import unittest
+
+from . import session
+from .. import lib
+from .. import paths
+from .. import test
+
+class Test_Imiscsvrinfo(session.make_sessions_mixin([('otherrods', 'rods')], [('alice', 'apass')]), unittest.TestCase):
+
+    def setUp(self):
+        super(Test_Imiscsvrinfo, self).setUp()
+        self.admin = self.admin_sessions[0]
+        self.user = self.user_sessions[0]
+
+    def tearDown(self):
+        super(Test_Imiscsvrinfo, self).tearDown()
+
+    @unittest.skipUnless(test.settings.USE_SSL, 'Test case assumes use of SSL/TLS')
+    def test_imiscsvrinfo_prints_ssl_info_with_ssl_enabled__issue_7986(self):
+        _, out, _ = self.user.assert_icommand(['imiscsvrinfo'], 'STDOUT')
+        expected_outputs = ['enabled: true', 'issuer_name', 'not_after', 'not_before', 'public_key', 'subject_alternative_names', 'subject_name']
+        for output in expected_outputs:
+            self.assertIn(output, out);
+
+    @unittest.skipIf(test.settings.USE_SSL, 'Test case assumes absence of SSL/TLS')
+    def test_imiscsvrinfo_prints_ssl_info_with_ssl_disabled__issue_7986(self):
+        self.user.assert_icommand(['imiscsvrinfo'], 'STDOUT', 'enabled: false')
```

Comment:
Let's add a few assertions about the negative space. For example, assert `issuer_name` and `not_after` aren't in the output.

The list doesn't have to span all properties though.